### ISSUE TITLE
[Test🧪] Complete NameServer request classifier and transport policy tests

### ISSUE THREAD

**Issue by mxsm** [mxsm/rocketmq-rust#10165]
### Prerequisites

- [x] I have searched existing test issues

### Test Scope

Module: rocketmq-namesrv
Files: rocketmq-namesrv/src/security.rs

`classify_namesrv_request` enumerates the full NameServer request surface, but the existing tests only assert 8 of the roughly 20 classified codes. Every enum arm should be covered so a future code addition or reclassification is caught. The `NameServerTransportPolicy` implementations of `IngressPolicy` and `RequestPolicy` also have no tests, even though the fail-closed boundary for unknown codes is a security property.

Where to start: extend the `mod tests` in `src/security.rs`. For the policy tests, build a `rocketmq_security_api::SecurityRequestView::new(code, version, &HashMap::new(), None, None)` and call `NameServerTransportPolicy.evaluate_ingress(...)`. For the authenticated path use `rocketmq_security_api::evaluate_request(&policy, &RequestContext::new(...))` with a `Principal` from `rocketmq_security_api`.

Validation commands (run from the repository root):

```
cargo test -p rocketmq-namesrv security
cargo fmt -p rocketmq-namesrv -- --check
cargo clippy --workspace --no-deps --all-targets --all-features -- -D warnings
```

### Test Cases

- [ ] Extend the existing expected-table test to cover every code currently classified, including the ones not yet asserted: `GetKvConfig`, `QueryDataVersion`, `GetBrokerMemberGroup`, `GetKvlistByNamespace`, `GetTopicsByCluster`, `GetSystemTopicListFromNs`, `GetUnitTopicList`, `GetHasUnitSubTopicList`, `GetHasUnitSubUnunitTopicList`, `GetAllTopicListFromNameserver` (AdminRead) and `DeleteKvConfig`, `WipeWritePermOfBroker`, `AddWritePermOfBroker`, `DeleteTopicInNamesrv`, `RegisterTopicInNamesrv` (AdminWrite).
- [ ] `NameServerRequestClass::as_str` returns the documented label for all four variants.
- [ ] `evaluate_ingress` returns `Ok(IngressDecision::AllowToContinue)` for a classified code and `Ok(IngressDecision::Deny)` for a foreign code such as `RequestCode::PullMessage`.
- [ ] `evaluate_request` with a present principal returns `AuthorizationDecision::Allow` for a classified code and `AuthorizationDecision::Deny(AuthorizationDenial::PermissionDenied)` for a foreign code.
- [ ] `evaluate_request` with no principal returns `AuthorizationDecision::Deny(AuthorizationDenial::SubjectUnknown)` without invoking the policy.
- [ ] Existing security tests keep passing unchanged.

### Willing to contribute?

- [ ] I can submit a PR for this
